traits/tests/test_category.py

```diff
@@ -104,3 +104,20 @@ class are not inherited.
         except SystemError:
             pass
         return
+
+    def test_subclasses_dont_modify_category_base_class(self):
+        # Regression test for enthought/traits#507
+        from traits.api import Category, HasTraits, Str
```

Comment:
Does this need to be imported lazily? It looks like it's already at the top of the module.

Reply:
No, it doesn't. Copy and paste error. Will fix.

Reply:
Fixed.